code/datums/gamemode/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -914,11 +914,11 @@
 	restricted_from_jobs = list()
 	enemy_jobs = list("Warden","Head of Security")
 	required_enemies = list(1,1,1,1,1,1,1,1,1,1)
-	required_pop = list(0,0,10,10,15,15,20,20,20,25)
+	required_pop = list(25,20,20,20,15,15,10,10,0,0)
 	required_candidates = 1
-	weight = 3
+	weight = 1
```

Comment:
Initial PR had weight = 1. It was tripled due to Prisoner not firing often. I believe that Prisoner didn't fire often because of the backwards requirements, so I reverted the PR to buff the weight.